### PR TITLE
[receiver/kubeletstats] Add note about metrics deprecation on specific release

### PR DESCRIPTION
**Pull request by ChrsMark**
**Description:** <Describe what has changed.>

Addresses https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27885#issuecomment-2275386899.

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by mx-psi — APPROVED** [review]

**Comment by ChrsMark** [comment]
Note this is just my suggestion for the target release. We should wait for confirmation at https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27885#issuecomment-2283266028.

File: receiver/kubeletstatsreceiver/README.md [hunk 1]
```diff
@@ -285,3 +285,18 @@ rules:
...
+
+The following metrics will be removed in `v0.111.0` version:
+- `k8s.node.cpu.utilization` (replaced by `k8s.node.cpu.usage`)
+- `k8s.pod.cpu.utilization` (replaced by `k8s.pod.cpu.usage`)
+- `container.cpu.utilization` (replaced by `container.cpu.usage`)
+
+Users can use the following metrics instead:
+- `k8s.node.usage`
+- `k8s.pod.usage`
+- `container.usage`
```
Comment (crobert-1) [inline receiver/kubeletstatsreceiver/README.md:299]: Out of ignorance, what's the relevance of this section? The comment above shares which metrics are being removed, and what metric they're being replaced by. This appears to be somewhat duplicating the "replaced by" message, but with different metrics. (I very well may be missing something, just want to check.)

Comment (ChrsMark) [reply receiver/kubeletstatsreceiver/README.md]: Good catch! Actually there was a `*.cpu.*` part missing by mistake there. Thank's for spotting this!

File: receiver/kubeletstatsreceiver/README.md [hunk 2]
```diff
@@ -285,3 +285,18 @@ rules:
...
+- `k8s.pod.cpu.utilization` (replaced by `k8s.pod.cpu.usage`)
+- `container.cpu.utilization` (replaced by `container.cpu.usage`)
+
+Users can use the following metrics instead:
+- `k8s.node.cpu.usage`
+- `k8s.pod.cpu.usage`
+- `container.cpu.usage`
+
+More information about the deprecation plan and
+the background reasoning can be found at https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27885.
```
Comment (TylerHelmuth) [inline receiver/kubeletstatsreceiver/README.md:302]: I like the idea of adding something that explicitly stated "this is a rename, the values are exactly the same as before, only the metric name has changed to be in alignment with the OpenTelemetry specification".

**Comment by ChrsMark** [comment]
Marked it as draft while we wait for a decision at https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27885.

**Pull request closed without merging**